### PR TITLE
Torch model

### PR DESCRIPTION
**Pull request by haraschax**

### CODE REVIEW

**Review by jyoung8607 — APPROVED** [review]
Mooted with 42e2481

File: selfdrive/car/volkswagen/interface.py [hunk 1]
```diff
@@ -69,10 +69,10 @@ def update(self, c, can_strings):
     # The camera CAN has no signals we use at this time, but we process it
     # anyway so we can test connectivity with can_valid
     self.cp.update_strings(can_strings)
-    self.cp_cam.update_strings(can_strings)
+    # self.cp_cam.update_strings(can_strings)
 
     ret = self.CS.update(self.cp)
-    ret.canValid = self.cp.can_valid and self.cp_cam.can_valid
+    ret.canValid = self.cp.can_valid
```
[the remark this answers is not in the document]
Comment (HaraldSchafer) [reply selfdrive/car/volkswagen/interface.py]: Yeah was just for testing, already reverted in the PR, thanks!

**Comment by ShaneSmiskol** [comment]
Seems like there's a memory leak, it grows to ~70% usage and then freezes and reboots

**Comment by geohot** [comment]
This branch isn't ready yet!

**Comment by geohot** [comment]
Can I test on the drive home?

**Comment by mitchellgoffpc** [comment]
@geohot go for it!

**Comment by csouers** [comment]
UI crashes for me after going onroad. I'm on commit 87a42c1.

Here's some scrollback which I hope is useful:
```
type = 0x0004 = CL_DEVICE_TYPE_GPU                                                  
driverState: Reader was evicted, reconnecting                                       
driverState: Reader was evicted, reconnecting                                       
cameraOdometry: Reader was evicted, reconnecting                                    
loaded model with size: 56036227                                                    
thermald logmessaged ui uploader deleter updated logcatd tombstoned sensord pandad controlsd plannerd loggerd radard calibrationd paramsd camerad proclogd locationd cloc
ksd ubloxd dmonitoringd dmonitoringmodeld gpsd rtshield modeld                      
frontFrame: Reader was evicted, reconnecting                                                                                                                             
selfdrive/camerad/main.cc: client start fd 146                                      
driverState: Reader was evicted, reconnecting                                       
selfdrive/camerad/main.cc: client start fd 147                                      
selfdrive/camerad/main.cc: client start fd 148                                      
driverState: Reader was evicted, reconnecting                                       
loaded model with size: 1134208                                                     
ioctl returned -1 with errno 22                                                     
model: Reader was evicted, reconnecting                                             
thermald logmessaged ui uploader deleter updated logcatd tombstoned sensord pandad controlsd plannerd loggerd radard calibrationd paramsd camerad proclogd locationd cloc
ksd ubloxd dmonitoringd dmonitoringmodeld gpsd rtshield modeld                      
vipc_recv err: Connection reset by peer                                             
selfdrive/camerad/main.cc: client end fd 146                                        
model: lambda/div:0 -> outputs/concat:0                                             
input product is 307200                                                             
selfdrive/camerad/main.cc: client start fd 146                                      
selfdrive/loggerd/loggerd.cc: camera 0 rotate encoder to /data/media/0/realdata/2020-11-11--20-46-22--0.
selfdrive/loggerd/loggerd.cc: camera 1 rotate encoder to /data/media/0/realdata/2020-11-11--20-46-22--0.
selfdrive/modeld/dmonitoringmodeld.cc: connected with buffer size: 1492992   ```

**Comment by geohot** [comment]
Sorry, fixing now.

**Pull request merged**